### PR TITLE
Add stackdriver to symfony

### LINKED ISSUE #815: Add Stackdriver Logging/Error Reporting for Symfony on App Engine

### PR DESCRIPTION
**Pull request by bshaffer**
fixes #815

### CODE REVIEW

File: appengine/php72/symfony-framework/README.md [hunk 1]
```diff
@@ -205,6 +205,46 @@ database. This tutorial uses the database name `symfonydb` and the username
 
         gcloud app deploy
 
+## Set up Stackdriver Logging and Error Reporting
+
+Install some cloud libraries for Stackdriver integration
```
Comment (SurferJeffAtGoogle) [inline appengine/php72/symfony-framework/README.md]: Would you like me to work through the instructions and see if I can get it working myself?  (Would take a day or two.)

File: appengine/php72/laravel-framework/README.md [hunk 2]
```diff
@@ -133,15 +133,15 @@ Logging and Error Reporting:
```
Comment (SurferJeffAtGoogle) [inline appengine/php72/laravel-framework/README.md]: The `php artisan serve` command failed for me:
```
rennie@rennie:~/gitrepos/php-docs-samples/appengine/php72/laravel-framework$ composer.phar global require "laravel/installer"
Changed current directory to /usr/local/google/home/rennie/.config/composer
Using version ^2.0 for laravel/installer
./composer.json has been created
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 12 installs, 0 updates, 0 removals
  - Installing symfony/process (v4.2.2): Downloading (100%)
  - Installing symfony/polyfill-ctype (v1.10.0): Downloading (100%)
  - Installing symfony/filesystem (v4.2.2): Downloading (100%)
  - Installing symfony/polyfill-mbstring (v1.10.0): Downloading (100%)
  - Installing symfony/contracts (v1.0.2): Downloading (100%)
  - Installing symfony/console (v4.2.2): Downloading (100%)
  - Installing guzzlehttp/promises (v1.3.1): Loading from cache
  - Installing ralouphie/getallheaders (2.0.5): Downloading (100%)
  - Installing psr/http-message (1.0.1): Loading from cache
  - Installing guzzlehttp/psr7 (1.5.2): Downloading (100%)
  - Installing guzzlehttp/guzzle (6.3.3): Downloading (100%)
  - Installing laravel/installer (v2.0.1): Downloading (100%)
symfony/contracts suggests installing psr/cache (When using the Cache contracts)
symfony/contracts suggests installing psr/container (When using the Service contracts)
symfony/contracts suggests installing symfony/cache-contracts-implementation ()
symfony/contracts suggests installing symfony/service-contracts-implementation ()
symfony/contracts suggests installing symfony/translation-contracts-implementation ()
symfony/console suggests installing psr/log-implementation (For using the console logger)
symfony/console suggests installing symfony/event-dispatcher ()
symfony/console suggests installing symfony/lock ()
guzzlehttp/guzzle suggests installing psr/log (Required for using the Log middleware)
Writing lock file
Generating autoload files
rennie@rennie:~/gitrepos/php-docs-samples/appengine/php72/laravel-framework$ php artisan serve
Could not open input file: artisan
rennie@rennie:~/gitrepos/php-docs-samples/appengine/php72/laravel-framework$ ls
app  app-dbsessions.yaml  app.yaml  bootstrap  composer.json  config  phpunit.xml.dist  README.md  routes  test
rennie@rennie:~/gitrepos/php-docs-samples/appengine/php72/laravel-framework$
```

Comment (bshaffer) [reply appengine/php72/laravel-framework/README.md]: So you still need to call `laravel new [project]` and then if you cd in to that directory you'll be able to call ` php artisan serve` (see [here](https://laravel.com/docs/5.4/installation#installing-laravel)). 

But those steps are for the Laravel tutorial. I fixed a Laravel typo in this PR, hence the confusion, but this PR is for the step of adding Stackdriver integration to symfony, which is a different tutorial/README.

File: appengine/php72/symfony-framework/README.md [hunk 3]
```diff
@@ -205,6 +205,47 @@ database. This tutorial uses the database name `symfonydb` and the username
...
+cp src/EventSubscriber/ExceptionSubscriber.php \
+    /path/to/symfony/src/EventSubscriber
+```
+
+The two files needed are as follows:
+
+  1. [`config/packages/prod/monolog.yaml`](app/config/packages/prod/monolog.yaml) - Adds Stackdriver Logging to your Monolog configuration.
+  1. [`src/EventSubscriber/ExceptionSubscriber.php`](src/EventSubscriber/ExceptionSubscriber.php) - Event subscriber which sends exceptions to Stackdriver Error Reporting.
+
+If you'd like to test the logging and error reporting, you can also copy over `LoggingController.php`, which
```
[the remark this answers is not in the document]
Comment (bshaffer) [reply appengine/php72/symfony-framework/README.md]: This is a good idea, we would need to create a special repo for this however, which is a little more overhead. I'm not against it, but we could also make the copy commands easier, like:

```bash
export SYMFONY_DIR=/path/to/my-symfony-project
export SYMFONY_GCP_SAMPLES_DIR=/path/to/php-docs-samples/appengine/php72/symfony-framework
cp $SYMFONY_GCP_SAMPLES_DIR/file1.php $SYMFONY_DIR
cp $SYMFONY_GCP_SAMPLES_DIR/file2.php $SYMFONY_DIR
cp $SYMFONY_GCP_SAMPLES_DIR/file3.php $SYMFONY_DIR
```

File: appengine/php72/symfony-framework/README.md [hunk 4]
```diff
@@ -205,6 +205,47 @@ database. This tutorial uses the database name `symfonydb` and the username
...
+exposes the routes `/en/logging/notice` and `/en/logging/exception` for ensuring your logs are being sent to
+Stackdriver:
+
+  1. [`src/Controller/LoggingController.php`](src/Controller/LoggingController.php) - Controller for testing logging and exceptions.
+
```
[the remark this answers is not in the document]
Comment (bshaffer) [reply appengine/php72/symfony-framework/README.md]: Good call, I'll add links to the Error Reporting and Logging UI!

File: appengine/php72/symfony-framework/README.md [hunk 5]
```diff
@@ -205,6 +205,47 @@ database. This tutorial uses the database name `symfonydb` and the username
 
         gcloud app deploy
 
+## Set up Stackdriver Logging and Error Reporting
```
[the remark this answers is not in the document]
Comment (bshaffer) [reply appengine/php72/symfony-framework/README.md]: I was going to make a community article for it. Then we could link to it in the sidenav. What do you think?

File: appengine/php72/symfony-framework/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (SurferJeffAtGoogle) [reply appengine/php72/symfony-framework/README.md]: Yeah, this idea is definitely **not** blocking this PR.

File: appengine/php72/symfony-framework/README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (SurferJeffAtGoogle) [reply appengine/php72/symfony-framework/README.md]: That sounds good to me.  Also, not blocking this PR.

**Pull request merged**